lib/charms/kubernetes_charm_libraries/v0/multus.py

```diff
@@ -619,3 +627,15 @@ def _on_remove(self, event: RemoveEvent) -> None:
                 self.kubernetes.delete_network_attachment_definition(
                     name=network_attachment_definition.metadata.name  # type: ignore[union-attr]
                 )
+
+    def delete_pod(self) -> None:
+        """Delete the pod."""
+        self.kubernetes.delete_pod(self._pod)
```

Comment:
Would it be ok to just do
`self.kubernetes.client.delete(Pod, pod_name, namespace)`
?